server_environment/readme/DESCRIPTION.md

```diff
@@ -6,7 +6,7 @@ are stored in the `server_environment_files` companion module.
 
 The `server_environment_files` module is optional, the values can be set
 using an environment variable with a fallback on default values in the
-database.
+database. You will be able to overwrite some odoo options.
```

Comment:
Can all options be changed?
If only some options can be changed, it would help to document which ones.

Reply:
Yes all odoo options of the config file can be overwritten this way.

Reply:
Then I suggest this change:

```suggestion
database. You will be able to overwrite any Odoo options.
```

Reply:
IMO this sentence is confusing if we don't give extra details (with the key `server_environment_allow_overwrite_options_section`,...).
But all this is already explained in the "Configured" section.
I think it's better to keep it as it is with a simple description.